renderlib/graphics/gl/Util.cpp

```diff
@@ -337,6 +354,113 @@ BoundingBoxDrawable::drawLines(const glm::mat4& transform, const glm::vec4& colo
   _shader->release();
 }
 
+// physicalscale is max of physical dims x,y,z
+static float
+computePhysicalScaleBarSize(const float physicalScale)
+{
+  // note this result will always be some integer power of 10 independent of zoom...
+  return pow(10.0f, floor(log10(physicalScale / 2.0f)));
+}
```

Comment:
Copied in `ScaleBarTool.cpp`?

Reply:
- [ ] refactor this so it's not duplicated